.gitlab-ci.yml

```diff
@@ -598,12 +594,43 @@ release:deployment:tag:
       ./scripts/deploy-image.sh "${image[0]}" \'mainnet\' "$CI_REGISTRY_IMAGE";
       echo \'Deploying ECS service to mainnet\';
       '
+    - echo 'Waiting for seed nodes to update'
```

Comment:
Keep this consistent:

```
Waiting for Mainnet Deployment
```

You have to remove the previous message too.

Also this has to be done for `:branch` and `:tag`.

Confirm that these 2 are both running... if they both run, that means mainnet gets deployed on every commit.